mscr-ui/src/modules/crosswalk-editor/mappings-accordion/index.tsx

```diff
@@ -202,20 +216,40 @@ export default function JointListingAccordion(props: any) {
                 <span className="fw-bold ps-4">Target</span>
               </StyledTableCell>
               <StyledTableCell className="col-1"></StyledTableCell>
-              <StyledTableCell className="col-2"></StyledTableCell>
+              <StyledTableCell className="col-2">
+                <SearchWrapper className="w-100">
+                  <SearchInput
+                    labelText={props.caption}
```

Comment:
You're not actually passing caption in the props (which would be more apparent if props were typed). Rather set labelText, but use `labelMode="hidden"` to not have it visually present.